### PR TITLE
MIL-40 Basic configuration

### PR DESCRIPTION
**Pull request by matizaj**
Add:
webpack
babel, 
eslint 
prettier
sass 
and appropriate plugins for webpack

### CODE REVIEW

File: frontend/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb",
+        "prettier",
```
Comment (pawelkonior) [inline frontend/.eslintrc.js:8]: should be last in this list

Comment (matizaj) [reply frontend/.eslintrc.js]: yes sir, prettier moved in the end of this list

File: frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "url": "git+ssh://git@github.com/402-Code/medical-calculator.git"
+  },
+  "author": "402-Code",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/402-Code/medical-calculator/issues"
+  },
+  "homepage": "https://github.com/402-Code/medical-calculator#readme",
+  "dependencies": {
+    "@material-ui/core": "^4.12.3",
```
Comment (pawelkonior) [inline frontend/package.json:29]: depracated, use @mui/material (mui.com)

Comment (matizaj) [reply frontend/package.json]: done

File: frontend/.babelrc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "@babel/preset-react",
+    "@babel/preset-env"
+  ]
```
Comment (pawelkonior) [inline frontend/.babelrc:5]: "plugins": [
    [
      "@babel/plugin-transform-runtime",
      {
        "regenerator": true
      }
    ]
  ]

File: frontend/.eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb",
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "prettier",
+    ],
+    "presets": [
```
Comment (pawelkonior) [inline frontend/.eslintrc.js:12]: I am sure, but I think that this doesn`t work

File: frontend/package.json [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    "@mui/material": "^5.3.0",
+    "axios": "^0.24.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.16.7",
+    "@babel/preset-env": "^7.16.8",
+    "@babel/preset-react": "^7.16.7",
+    "babel-loader": "^8.2.3",
```
Comment (pawelkonior) [inline frontend/package.json:40]: "@babel/runtime": "^7.16.7",
"@babel/plugin-transform-runtime": "^7.16.8",

File: frontend/package.json [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/402-Code/medical-calculator/issues"
+  },
+  "homepage": "https://github.com/402-Code/medical-calculator#readme",
+  "dependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@mui/material": "^5.3.0",
+    "axios": "^0.24.0",
```
Comment (pawelkonior) [inline frontend/package.json:32]: 0.25.0

**Review by pawelkonior — APPROVED** [review]

**Pull request merged**